Testing/SystemTests/tests/analysis/SANSLoadTest.py

```diff
@@ -218,6 +217,47 @@ def _run_load(state, publish_to_cache, use_cached, move_workspace=False, beam_co
         # self.assertTrue(load_alg.isExecuted())
         return load_alg
 
+    def _check_that_sets_to_zero(self, workspace, move_info, comp_name=None):
```

Comment:
Is this function used anywhere

Reply:
Removed the test which uses it but forgot to remove that function